Description: ARM xfail tests
 These tests are expected to fail on ARM due to the following issue in Numpy:
 https://github.com/numpy/numpy/issues/8325
Author: Graham Inggs <ginggs@debian.org>
Bug-Debian: http://bugs.debian.org/904785
Forwarded: no
Last-Update: 2018-07-29

--- a/xarray/tests/__init__.py
+++ b/xarray/tests/__init__.py
@@ -45,6 +45,9 @@
     reason="expected failure on ARM",
 )
 
+import platform
+arm_xfail = pytest.mark.xfail(platform.machine() == 'aarch64' or
+    'arm' in platform.machine(), reason='expected failure on ARM')
 
 def _importorskip(
     modname: str, minversion: str | None = None
